packages/react-components/src/components/VideoTile.tsx

```diff
@@ -229,6 +261,27 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
   const canShowLabel = showLabel && (displayName || (showMuteIndicator && isMuted));
   const participantStateString = participantStateStringTrampoline(props, locale);
 
+  /* @conditional-compile-remove(pinned-participants) */
+  const optionsMenuItems = (menuItems: VideoTileMenuItems): IContextualMenuItem[] => {
+    const contextualMenuItems: IContextualMenuItem[] = [];
+    menuItems.forEach((item: VideoTileMenuItem) => {
+      contextualMenuItems.push({
+        key: item.key,
+        text: item.text,
+        onClick: item.onClick,
+        iconProps: item.icon
+      });
+    });
+
+    return contextualMenuItems;
+  };
+
+  const optionsIconStyles = mergeStyles({
+    minWidth: '2rem',
+    minHeight: '0rem'
+  });
+  const icon = (): JSX.Element => <MoreHorizontal20Filled key={'chatOnIconKey'} primaryFill="currentColor" />;
```

Comment:
What is the key `chatOnIconKey` related to here?

Reply:
Removed the key was not needed here